### PR TITLE
Adds compact_units(amount, unit) and uses it in multitool.

### PR DESCRIPTION
**Pull request by ondrej008**
:cl: Ondrej008
tweak: multitool now displays in W, kW, MW depending on the power
/:cl:

Looks nicer.

### CODE REVIEW

File: code/modules/power/powernet.dm [hunk 1]
```diff
@@ -96,4 +96,29 @@
...
+
+example:
+compact_units(50000,    "W") -> 50 kW
+compact_units(50000000, "W") -> 50 MW
+
+*/
+/proc/compact_units(amount, unit)
+	if(amount < 1000)
+		return "[amount] [unit]"
+	if(amount > 1000*1000*1000*1000) // more readable than 1000000000
```
Comment (Qustinnus) [inline code/modules/power/powernet.dm]: there might be defines for this

Comment (ondrej008) [reply code/modules/power/powernet.dm]: I changed it to use the power thingy **

File: code/modules/power/powernet.dm [hunk 2]
```diff
@@ -96,4 +96,29 @@
...
+
+example:
+compact_units(50000,    "W") -> 50 kW
+compact_units(50000000, "W") -> 50 MW
+
+*/
+/proc/compact_units(amount, unit)
+	if(amount < 1000)
+		return "[amount] [unit]"
+	if(amount > 1000**4) // more readable than 1000000000
```
Comment (Qustinnus) [inline code/modules/power/powernet.dm]: yeah but im still pretty sure theres defines

Comment (ondrej008) [reply code/modules/power/powernet.dm]: any idea where I should look? I tried misc.dm and math.dm in __DEFINES but nothing so far.

**Comment by Qustinnus** [comment]
eh w.e

**Pull request merged**